frontend/main.go

```diff
@@ -18,9 +17,7 @@ const ProgramName = "ARO HCP Frontend"
 var Version = "unknown" // overridden by Makefile
 
 func main() {
-	// FIXME Centralize logging options?
-	handlerOptions := slog.HandlerOptions{}
-	logger := slog.New(slog.NewJSONHandler(os.Stdout, &handlerOptions))
+	logger := DefaultLogger()
```

Comment:
I extracted that default logger creation to a function as it is instantiated from multiple places now.